storage/src/test/scala/io/delta/storage/integration/S3LogStoreUtilIntegrationTest.scala

```diff
@@ -0,0 +1,107 @@
+package io.delta.storage.integration
+
+import io.delta.storage.internal.S3LogStoreUtil
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.Path
+import org.apache.hadoop.fs.s3a.{S3AFileSystem, UploadInfo}
+import org.scalatest.Tag
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalactic.source
+
+import java.net.URI
+import java.nio.file.Paths
+import scala.math.max
+import scala.math.ceil
+import scala.math.round
+
+class S3LogStoreUtilIntegrationTest extends AnyFunSuite {
+  private val runIntegrationTests: Boolean =
+    Option(System.getenv("S3_LOG_STORE_UTIL_TEST_ENABLED")).exists(_.toBoolean)
+  private val bucket = System.getenv("S3_LOG_STORE_UTIL_TEST_BUCKET")
+  private val keyPrefix = System.getenv("S3_LOG_STORE_UTIL_TEST_KEY_PREFIX")
+  private lazy val fs: S3AFileSystem = {
+    val fs = new S3AFileSystem()
+    fs.initialize(new URI(s"s3a://$bucket"), configuration)
+    fs
+  }
+  private val configuration = new Configuration()
+  configuration.set( // for local testing only
+    "fs.s3a.aws.credentials.provider",
+    "com.amazonaws.auth.profile.ProfileCredentialsProvider"
+  )
+
+  private val file = Paths.get("/tmp/tmp.json").toFile
```

Comment:
this is giving me 
```
[info] - setup delta logs *** FAILED ***
[info]   com.amazonaws.SdkClientException: Unable to calculate MD5 hash: /tmp/tmp.json (No such file or directory)
[info]   at com.amazonaws.services.s3.AmazonS3Client.getInputStream(AmazonS3Client.java:1897)
[info]   at com.amazonaws.services.s3.AmazonS3Client.uploadObject(AmazonS3Client.java:1815)
[info]   at com.amazonaws.services.s3.AmazonS3Client.putObject(AmazonS3Client.java:1794)
[info]   at com.amazonaws.services.s3.transfer.internal.UploadCallable.uploadInOneChunk(UploadCallable.java:168)
[info]   at com.amazonaws.services.s3.transfer.internal.UploadCallable.call(UploadCallable.java:148)
[info]   at com.amazonaws.services.s3.transfer.internal.UploadMonitor.call(UploadMonitor.java:115)
[info]   at com.amazonaws.services.s3.transfer.internal.UploadMonitor.call(UploadMonitor.java:45)
[info]   at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[info]   at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[info]   at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
```

Reply:
Forgot to include the sample file. It should be fixed now.